perceval/converters/abstract_converter.py

```diff
@@ -102,7 +104,35 @@ def apply_input_state(self):
     def convert(self, gate_circuit, use_postselection: bool = True) -> Processor:
```

Comment:
Shouldn't `convert` not be abstract after the refactoring? Can't it be generic ? And only `_generate_gate_sequence` specific?